src/js/components/stock-movement-wizard/outbound/AddItemsPage.jsx

```diff
@@ -1109,7 +1109,8 @@ class AddItemsPage extends Component {
                   }}
                   className="btn btn-outline-primary btn-form float-right btn-xs"
                   disabled={
-                    !_.some(values.lineItems, item => !_.isEmpty(item)) || invalid || showOnly
+                    !_.some(values.lineItems, item => !_.isEmpty(item)) || (values.lineItems.length === 1 && !('product' in values.lineItems[0])) ||
```

Comment:
I think this part: `values.lineItems, item => !_.isEmpty(item))` is not required anymore.